src/objects.ts

```diff
@@ -27,8 +27,12 @@ function createBook(
   genre: string
 ): Book {
   // write your code here...
-
-  return {} as Book; // replace "{} as Book" with what you see is fit
+  return {
+    title: title,
+    author: author,
+    publishedYear: publishedYear,
+    genre: genre,
+  }; // replace "{} as Book" with what you see is fit
```

Comment:
A fun thing you can do in JS: if the key and the variable share the same name you can do:
```ts
return {
  title,
  author,
  publishedYear,
  genre
};
```